src/main/java/bruno/Storage.java

```diff
@@ -0,0 +1,138 @@
+package bruno;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+import bruno.exceptions.BrunoException;
+import bruno.exceptions.FileLoadingException;
+import bruno.task.Deadline;
+import bruno.task.Event;
+import bruno.task.Task;
+import bruno.task.ToDo;
+
+/**
+ * The Storage class handles the reading and writing of task data
+ * to and from a file. It ensures the directory and file for task
+ * data storage exist and provides methods to load tasks from a file
+ * and update the file with the current task list.
+ */
+public class Storage {
+    private DateTimeFormatter formatter2 = DateTimeFormatter.ofPattern("MMM d yyyy HH:mm");
```

Comment:
Perhaps the variable name could be `formatter` or `dateTimeFormatter`? Since there is no other formatter within this file.